validator.go

```diff
@@ -13,13 +13,14 @@ import (
 	"net"
 	"net/url"
 	"reflect"
-	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
 	"unicode"
 	"unicode/utf8"
+
+	"github.com/h2so5/goback/regexp"
```

Comment:
The Golang regexp package does not provide support for negative lookbehinds which is required in order to correctly split strings on commas that are not preceded by a backslash.